pkg/rancher-desktop/backend/lima.ts

```diff
@@ -781,14 +819,20 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
 
     try {
       // Print a slightly different message if execution fails.
-      const stdout = await this.limaWithCapture(true, 'shell', '--workdir=.', MACHINE_NAME, ...command);
+      const stdout = await this.limaWithCapture({ expectFailure: true }, 'shell', '--workdir=.', MACHINE_NAME, ...command);
 
       if (options.capture) {
         return stdout;
       }
-    } catch (ex) {
+    } catch (ex: any) {
       if (!expectFailure) {
         console.log(`Lima: executing: ${ command.join(' ') }: ${ ex }`);
+        if (this.debug && 'stdout' in ex) {
+          console.error(ex.stdout);
```

Comment:
Can you add an indication so we know which output is from stdout and which from stderr?

Reply:
Done in f363bc2f175535a251c9912abc3084b467b36439